src/functions-and-arrays.js

```diff
@@ -52,14 +111,27 @@ const wordsUnique = [
   'bring'
 ];
 
-function uniquifyArray() {}
+function uniquifyArray(arr) {
+  let noDuplicates = [];
+   
+return noDuplicates;
+}
```

Comment:
Did you read about `indexOf() `and still not understand how to apply it?

- the syntax of this method looks like this arrayName.indexOf(searchElement).
- you want to access array, then call the indexOf method and pass an element you are checking as argument
- indexOf() method returns and index of an element (inside that array), or if it is not existant  -1